fedimint-server/src/config/distributedgen.rs

```diff
@@ -210,30 +229,15 @@ impl<G: DkgGroup> Dkg<G> {
         for element in poly {
             engine
                 .write_all(element.to_bytes().as_ref())
-                .expect("hashes");
+                .expect("Writing to a hash engine cannot fail.");
         }
 
         Sha256::from_engine(engine)
     }
+}
 
-    fn broadcast(&self, message: &DkgMessage<G>) -> DkgStep<G> {
-        DkgStep::Messages(
-            self.num_peers
-                .peer_ids()
-                .filter(|peer| *peer != self.identity)
-                .map(|peer| (peer, message.clone()))
-                .collect(),
-        )
-    }
-
-    /// Get a second generator by hashing the first one to the curve
-    fn gen_h(&self) -> G {
-        let mut hash_engine = sha3::Sha3_256::new();
-
-        hash_engine.update(self.generator.clone().to_bytes().as_ref());
-
-        G::random(&mut ChaChaRng::from_seed(hash_engine.finalize().into()))
-    }
+fn gen_h<G: DkgGroup>() -> G {
+    G::random(&mut ChaChaRng::from_seed([42; 32]))
```

Comment:
Can we just keep hashing, but use sha256 to get rid of that pretty random sha3 dep? Hashing the main generator is a very standard construction: https://bitcoin.stackexchange.com/a/116733